.github/workflows/create-demo-clusters.yml

```diff
@@ -344,6 +398,103 @@ jobs:
           ./scale/launch_workload.sh np-load
           echo "Fake workload has been deployed to the long-running cluster" >> "$GITHUB_STEP_SUMMARY"
 
+  start-secured-cluster:
+    name: Start secured cluster
+    needs: [properties, start-acs, create-long-running-secured-cluster]
+    runs-on: ubuntu-latest
+    env:
+      SECURED_CLUSTER_NAME: ${{ inputs.secured-cluster-name }}
+      TAG: ${{github.event.inputs.version}}
+      KUBECONFIG: artifacts/kubeconfig
+      INFRA_TOKEN: ${{secrets.INFRA_TOKEN}}
+      USE_GKE_GCLOUD_AUTH_PLUGIN: "True"
+      ROX_ADMIN_PASSWORD: ${{ needs.start-acs.outputs.rox_password }}
+      ROX_ADMIN_USERNAME: admin
+      CENTRAL_IP: ${{ needs.start-acs.outputs.central-ip }}
+    steps:
+      - uses: stackrox/actions/infra/install-infractl@v1
+      - name: Test readiness for central
```

Comment:
This is about cluster, not the central itself

Reply:
Done